documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -135,8 +135,21 @@ spec:
 EOF
 ----
 <1> Specify the name of the VMware `Provider` CR.
-<2> Specify the managed object reference (MOR) of the VMware host.
+<2> Specify the managed object reference (MOR) of the VMware host. To retrieve the name of the MOR, see the note that follows.
```

Comment:
we refer to MOR also in other places, so I think we should have this before

Reply:
@ahadas The Host manifest is the first place we refer to MOR, but, of course you're right, we refer to it elsewhere (later).  Do you want me to a) move the note to before the procedure b) include the note for all manifests where MOR  is mentioned c) write "To retrieve the MOR, see..." for the other mentions, without copy-pasting the note.

Reply:
I think that moving it before the procedure makes more sense

Reply:
Moved the new section to before the migration procedure.